src/org/thoughtcrime/securesms/service/RegistrationService.java

```diff
@@ -260,17 +260,16 @@ private void handleCommonRegistration(TextSecureAccountManager accountManager, S
     DatabaseFactory.getIdentityDatabase(this).saveIdentity(self.getRecipientId(), identityKey.getPublicKey());
     DirectoryHelper.refreshDirectory(this, accountManager, number);
 
-    //if (GooglePlayServicesUtil.isGooglePlayServicesAvailable(this) == ConnectionResult.SUCCESS &&
-    //    !BuildConfig.FORCE_WEBSOCKETS)
-    //{
-    //  RedPhoneAccountManager redPhoneAccountManager = new RedPhoneAccountManager(BuildConfig.REDPHONE_MASTER_URL,
-    //                                                                           new RedPhoneTrustStore(this),
-    //                                                                           number, password);
-    //  String verificationToken = accountManager.getAccountVerificationToken();
-    //  redPhoneAccountManager.createAccount(verificationToken, new RedPhoneAccountAttributes(signalingKey,
-    //          TextSecurePreferences.getGcmRegistrationId(this)));
-    //}
-    accountManager.getAccountVerificationToken();
+    String verificationToken = accountManager.getAccountVerificationToken();
```

Comment:
> So @JavaJens Could we get this integrated here at first?

The consensus here seems to be that this (JavaJens) repository shouldn't diverge much from upstream because there is still hope that Moxie will some day merge it back. Unfortunately I have given up this hope, Moxie is extremely hostile to anything that is not distributed by him on Google Play (and websocket support goes directly against his "walled garden" wishes).